scripts/vet.sh

```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Check for tabs in Markdown files
+not git grep $'\t' -- '*.md' | fail_on_output
```

Comment:
Why pipe to `fail_on_output`?  Inverting the result of `grep` should be sufficient?  I.e. `grep` exits with 0 if there is a match and 1 if there is not.

Reply:
Done